### ISSUE TITLE
"prerelease" policy?

### ISSUE THREAD

**Issue by mitchblank** [kulp/tenyr#73]
Since 2013 the homebrew package manager has included a formula for installing `tenyr` (Homebrew/homebrew#17433) and it has tracked releases all the way up to the current 0.9.7 (https://github.com/Homebrew/homebrew-core/pull/45147)

However, as a matter of policy homebrew generally does not allow the inclusion of versions considered "prerelease", since it's supposed to be a repository of "stable" versions.  This year there have been changes made to reject attempts to pre-build binaries ("bottling") if the source is a github release that the developer has marked as a pre-release (see https://github.com/Homebrew/brew/issues/7177)

Looking at https://github.com/kulp/tenyr/releases it seems that effectively *all* tenyr releases are considered "pre".  Homebrew has a system for specifying exceptions to its automated rules, but the maintainers want further explanation of why it's tracking pre-release software -- see https://github.com/Homebrew/homebrew-core/pull/68011#issuecomment-752405478

Is there a non-prerelease on the horizon?  Or maybe it's easier to just drop tenyr from homebrew entirely... analytics show it's not a widely installed homebrew formula at all:
```
$ brew info tenyr
[...]
install: 10 (30 days), 14 (90 days), 22 (365 days)
install-on-request: 10 (30 days), 14 (90 days), 22 (365 days)
```

**Comment by kulp** [comment]
@mitchblank, thank you for your inquiry.

From my perspective, it is true that all tenyr releases up to this point have been prereleases, insofar as I have not made any stability guarantees at any time. I do intend to publish [a stable v1.0](https://github.com/kulp/tenyr/milestone/1), but not until late 2021 (the ten-year anniversary of tenyr's inception). That will be, from my perspective, the first "non-pre" release. In my recollection, tenyr was added to Homebrew early enough that the [notability guidelines](https://docs.brew.sh/Acceptable-Formulae), which later would have excluded it, did not apply, or perhaps did not even exist.

tenyr is definitely a niche piece of software, and as much as it pains my ego to say it, I think the right thing is to have tenyr removed from `homebrew-core` and for me to [maintain my own tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap). I will comment in Homebrew/homebrew-core#68011 to that effect.

**Comment by kulp** [comment]
In https://github.com/Homebrew/homebrew-core/pull/68011#issuecomment-753483701 it seems like this issue might have been resolved by my changing recently-released [v0.9.8](https://github.com/kulp/tenyr/releases/tag/v0.9.8) to be a "regular" (non-pre) release. In the future, I will reserve "prerelease" to mean "post v1.0 release that has unusually low stability guarantees."

Thanks for bringing this to my attention, @mitchblank.
